tests/vmi_multus_test.go

```diff
@@ -871,14 +879,76 @@ var _ = Describe("SRIOV", func() {
 	})
 })
 
+var _ = Describe("Macvtap", func() {
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+	macvtapNetworkName := "net1"
+
+	createCirrosVMIWithMacvtapStaticIP := func(virtClient kubecli.KubevirtClient, networkName string, ifaceName string, ipCIDR string, mac *string) *v1.VirtualMachineInstance {
+		vmi := tests.NewRandomVMIWithMacvtapInterfaceEphemeralDiskAndUserdata(tests.ContainerDiskFor(tests.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n", networkName)
+		if mac != nil {
+			vmi.Spec.Domain.Devices.Interfaces[0].MacAddress = *mac
+		}
+		_, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+		Expect(err).ToNot(HaveOccurred())
+
+		// configure the client VMI
+		tests.WaitUntilVMIReady(vmi, tests.LoggedInCirrosExpecter)
+		configInterface(vmi, ifaceName, ipCIDR, "\\$ ", true)
+		return vmi
+	}
+
+	tests.BeforeAll(func() {
+		// provision macvtap NAD
+		result := virtClient.RestClient().
+			Post().
+			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, macvtapNetworkName)).
+			Body([]byte(fmt.Sprintf(macvtapNetworkConf, macvtapNetworkName, tests.NamespaceTestDefault, macvtapNetworkName))).
+			Do()
+		Expect(result.Error()).NotTo(HaveOccurred())
```

Comment:
I'm sure it just returns some gibberish on failure. Please add an expectation string describing what you want to happen